src/initialization/InitDistribution.cpp

```diff
@@ -22,13 +22,17 @@
 #include <AMReX_ParmParse.H>
 #include <AMReX_Print.H>
 
+#include <cmath>
 #include <string>
```

Comment:
For `std::runtime_error`:
```suggestion
#include <stdexcept>
#include <string>
```